homeassistant/components/rest_command/services.yaml

```diff
@@ -0,0 +1,62 @@
+# Describes the format for RESTful Command services
+
+service_name:
+  name: Service Name
+  description: >-
+    The name used to expose the service. E.g., in the above example, it would be ‘rest_command.example_request’.
+  fields:
+    url:
```

Comment:
These fields are not the service fields for this service. It seems like you have added the configuration schema to this service description.

The service call of the rest_command allows for passing payload data that can be used in templates.